searchcore/src/vespa/searchcore/proton/attribute/filter_attribute_manager.cpp

```diff
@@ -200,6 +200,18 @@ FilterAttributeManager::asyncForEachAttribute(std::shared_ptr<IAttributeFunctor>
     }
 }
 
+void
+FilterAttributeManager::asyncForAttribute(const vespalib::string &name, std::shared_ptr<IAttributeFunctor> func) const {
+    AttributeGuard::UP attr = _mgr->getAttribute(name);
+    if (!attr) { return; }
+    search::ISequencedTaskExecutor &attributeFieldWriter = getAttributeFieldWriter();
+    attributeFieldWriter.execute(attributeFieldWriter.getExecutorId((*attr)->getNamePrefix()),
```

Comment:
using and moving 'attr' in the same statement is not good

Reply:
Fixed